environment.yml

```diff
@@ -1,7 +1,6 @@
-name: sus-env2
+name: sus-env3
```

Comment:
What is `environment.yml` for anyway? Is this still Conda garbage? I thought I got rid of that for PyPy

Did you need to use this to install? It would be nice if this worked without dependencies, but if we need dependencies, maybe moving to `poetry`, that's what I did last time I modernized Python code

Clarifying the installation would be good, since we'll have to change that when we resetup the server environment to run this with python 3

Reply:
I assumed you were using `conda` to get Python 2 running, but I can get PyPy set up. We will need dependencies because it uses `js2py` to grab some data from Pokemon Showdown.

Reply:
@Marty-D do you remember the deal with this? I feel like I got rid of this dependency at some point

Reply:
Maybe I didn't get rid of it. Either way I don't know how to use Conda. We should have instructions on how to use it in the README. We don't have to switch to something else in this PR if it works OK.

Reply:
Re: PyPy, my understanding is that it's not as useful with recent versions of Python now that it has it's own JIT. So just having it work in whatever configuration is used on the server is fine. I'm not sure my changes ever got used in the main workflow anyway.

Reply:
> @Marty-D do you remember the deal with this? I feel like I got rid of this dependency at some point

I'm not sure, I wanted to use your changes on the new sim server but couldn't find how to configure PyPy on nix, or if I even had the right permissions to do so. Didn't have time to waste getting sh or whoever to do it so I just used the old conda stuff, which I feel like ended up taking longer to figure out than in 2019.

```
mkdir -p ~/miniconda3
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh
bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
rm -rf ~/miniconda3/miniconda.sh
~/miniconda3/bin/conda init bash
git clone https://github.com/Marty-D/Smogon-Usage-Stats
cd Smogon-Usage-Stats/
conda env create
tmux attach
cd Smogon-Usage-Stats/
conda env create
cd ..
source ~/.bashrc
conda --version
cd Smogon-Usage-Stats/
tmux attach
  source activate sus-env2
  conda info --envs
    # conda environments:
    #
    base                     /home/marty/miniconda3
    sus-env2              *  /home/marty/miniconda3/envs/sus-env2
```

Reply:
> Maybe I didn't get rid of it. Either way I don't know how to use Conda. We should have instructions on how to use it in the README. We don't have to switch to something else in this PR if it works OK.

This can happen in a different PR then.